javascript/javascript1/week3/SmartPhoneAppUsage.js

```diff
@@ -0,0 +1,70 @@
+let activities = [];
+function today() {
+  let today = new Date();
+  let dd = String(today.getDate()).padStart(2, "0");
+  let mm = String(today.getMonth() + 1).padStart(2, "0"); //January is 0!
+  let yyyy = today.getFullYear().toString().substr(-2);
+
+  return dd + "/" + mm + "-" + yyyy;
+}
+function addActivity(date, activity, duration) {
+  // the function will add activies object to activities array
+  activities.push({ date, activity, duration });
+}
+function showStatus(activitiesObject) {
```

Comment:
the parameter name looks like an object but it is array of objects, so I recommend calling this parameter `activities`

Reply:
changed parameter name

Reply:
I don't see the changes. anyway I'll approve for now.